docs/core/tools/dotnet-new.md

```diff
@@ -54,23 +54,23 @@ The command calls the [template engine](https://github.com/dotnet/templating) to
 |----------------------------------------------|---------------------------------|--------------|---------------------------------------|------------|
 | Console Application                          | [console](#console)             | [C#], F#, VB | Common/Console                        | 1.0        |
 | Class library                                | [classlib](#classlib)           | [C#], F#, VB | Common/Library                        | 1.0        |
-| WPF Application                              | [wpf](#wpf)                     | [C#]         | Common/WPF                            | 3.0        |
-| WPF Class library                            | [wpflib](#wpf)                  | [C#]         | Common/WPF                            | 3.0        |
-| WPF Custom Control Library                   | [wpfcustomcontrollib](#wpf)     | [C#]         | Common/WPF                            | 3.0        |
-| WPF User Control Library                     | [wpfusercontrollib](#wpf)       | [C#]         | Common/WPF                            | 3.0        |
-| Windows Forms (WinForms) Application         | [winforms](#winforms)           | [C#]         | Common/WinForms                       | 3.0        |
-| Windows Forms (WinForms) Class library       | [winformslib](#winforms)        | [C#]         | Common/WinForms                       | 3.0        |
+| WPF Application                              | [wpf](#wpf)                     | [C#], VB     | Common/WPF                            | 3.0        |
+| WPF Class library                            | [wpflib](#wpf)                  | [C#], VB     | Common/WPF                            | 3.0        |
+| WPF Custom Control Library                   | [wpfcustomcontrollib](#wpf)     | [C#], VB     | Common/WPF                            | 3.0        |
+| WPF User Control Library                     | [wpfusercontrollib](#wpf)       | [C#], VB     | Common/WPF                            | 3.0        |
+| Windows Forms (WinForms) Application         | [winforms](#winforms)           | [C#], VB     | Common/WinForms                       | 3.0        |
+| Windows Forms (WinForms) Class library       | [winformslib](#winforms)        | [C#], VB     | Common/WinForms                       | 3.0        |
```

Comment:
The versions in this table should have mentioned that VB is only supported starting with 5.0

Reply:
Indeed, that would be a good callout.